### ISSUE TITLE
Add startup message

### ISSUE THREAD

**Issue by ghost** [KusoIDE/KusoIDE#15]
add a start up message to kuso and add documentation and other useful links

**Comment by lxsameer** [comment]
I'm thinking about creating a splash screen like the one which emacs provided by default, and add some extra functionality like creating a new project or opening exists projects and so on

**Comment by ghost** [comment]
Yes I'm talking about that, but it should have some links and some documents, project home page and any other useful page/link.

**Comment by lxsameer** [comment]
Agree, But the hard part is to create this kind of screen. font-lock is a bit hard for me

**Comment by ghost** [comment]
I don't think so ;)
It should be easy.
